packages/block-library/src/paragraph/block.json

```diff
@@ -41,6 +41,7 @@
 				"text": true
 			}
 		},
+		"connections": true,
```

Comment:
Should we use `__experimentalConnections` or something similar? If I am not mistaken, that's being used for other attributes that are experimental, right?